Library/Homebrew/cask/lib/hbc/dsl.rb

```diff
@@ -207,13 +205,7 @@ def sha256(arg = nil)
     end
 
     def license(arg = nil)
-      return @license if arg.nil?
-      assert_only_one_stanza_allowed :license, !arg.nil?
-      @license ||= begin
-        DSL::License.new(arg) unless arg.nil?
-      rescue StandardError => e
-        raise CaskInvalidError.new(token, e)
-      end
+      odeprecated "Hbc::DSL#license"
```

Comment:
@reitermarkus After this is merged and tagged, I don’t foresee even a day passing before `license` is removed from all official taps.